src/angularjs/src/app/places/compare/compare.controller.js

```diff
@@ -28,9 +30,23 @@
                 return Places.getPlace(uuid);
             });
 
+            // first request score metadata
+            promises.unshift(ScoreMetadata.query().$promise);
```

Comment:
Now that we're using this in multiple places, can you add the `cache: true` parameter for the `ScoreMetadata.query` declaration on the `$resource`?

See: https://docs.angularjs.org/api/ngResource/service/$resource

Search for cache on the page at that link, its one of the options to the `actions` argument.